Source/Core/VideoBackends/OGL/PostProcessing.cpp

```diff
@@ -25,7 +25,7 @@ static const char s_vertex_shader[] = "out vec2 uv0;\n"
                                       "void main(void) {\n"
                                       "	vec2 rawpos = vec2(gl_VertexID&1, gl_VertexID&2);\n"
```

Comment:
I think we should use `(gl_VertexID&2) >> 1` instead.

Reply:
+1. Also implicit int->float conversion, is this allowed in GLES?

Reply:
What is VertexID, why are we bitmasking it and how does the change @degasus gave benefit us?

My original concern with this suggestion was that this code was not even changed in this PR.  It feels like bad practice to make an unrelated change in a PR.. I know others have done it but it never sits well with me.  In particular, a PR this big seems like this change could easily be lost.

If you both want me to do this, I'll happily open a new PR.

Reply:
I'm fine with making this change in a separate PR. Ideally we should move these shaders to videocommon anyway ;)